src/main/java/bio/terra/service/filedata/FSFile.java

```diff
@@ -99,6 +102,24 @@ public FSFile loadTag(String loadTag) {
     return this;
   }
 
+  // TODO: once metadata is collocated with the underlying cloud platform, this can be set
+  // explicitly
+  public CloudPlatform getCloudPlatform() {
+    try {
+      URI url = new URI(getGspath());
```

Comment:
Seems like we should rename the method `getGcspath()` now that it supports both gcs and azure paths.

Reply:
Unfortunately we can't do this easily since that field is deserialized from the object as it lives in Firestore.  Renaming a field there would require a Firestore migration, which I don't think is worth the rename (especially given this is an internal field)